content/en/docs/tasks/access-application-cluster/access-cluster.md

```diff
@@ -78,6 +78,8 @@ $ curl http://localhost:8080/api/
 
 Use `kubectl describe secret...` to get the token for the default service account:
 
+Using `grep/cut` approach:
```

Comment:
@chenrui333, looks good. A few nits:
- You may want to combine and reword, `Using grep/cut approach` with the `kubectl describe` instructions as the jsonpath approach does not use `kubectl describe secret`.

Use `kubectl describe secret` with grep/cut:

Use `jsonpath`: